sympy/physics/units/definitions.py

```diff
@@ -83,87 +100,133 @@
+                                  4 * pi / 10**7 * newton / ampere**2)
 # electric constat:
-e0 = electric_constant = vacuum_permittivity = Quantity("vacuum_permittivity", capacitance/length, 1/(u0 * c**2))
+e0 = electric_constant = vacuum_permittivity = Quantity(
+    "vacuum_permittivity", capacitance / length, 1 / (u0 * c**2))
 # vacuum impedance:
 Z0 = vacuum_impedance = Quantity("vacuum_impedance", impedance, u0 * c)
 # Coulomb's constant:
-coulomb_constant = electric_force_constant = Quantity("coulomb_constant", force*length**2/charge**2, 1/(4*pi*vacuum_permittivity), "k_e")
+coulomb_constant = electric_force_constant = Quantity(
+    "coulomb_constant", force * length**2 / charge**2,
+    1 / (4 * pi * vacuum_permittivity), "k_e")
 
-atmosphere = atmospheres = atm = Quantity("atmosphere", pressure, 101325 * pascal, "atm")
+atmosphere = atmospheres = atm = Quantity("atmosphere", pressure,
+                                          101325 * pascal, "atm")
 
-kPa = kilopascal = Quantity("kilopascal", pressure, kilo*Pa, "kPa")
-bar = bars = Quantity("bar", pressure, 100*kPa, "bar")
+kPa = kilopascal = Quantity("kilopascal", pressure, kilo * Pa, "kPa")
+bar = bars = Quantity("bar", pressure, 100 * kPa, "bar")
 pound = pounds = Quantity("pound", mass, 0.45359237 * kg)  # exact
-psi = Quantity("psi", pressure, pound * gee / inch ** 2)
+psi = Quantity("psi", pressure, pound * gee / inch**2)
 dHg0 = 13.5951  # approx value at 0 C
-mmHg = torr = Quantity("mmHg", pressure, dHg0 * acceleration_due_to_gravity * kilogram / meter**2)
-mmu = mmus = milli_mass_unit = Quantity("milli_mass_unit", mass, atomic_mass_unit/1000)
+mmHg = torr = Quantity(
+    "mmHg", pressure, dHg0 * acceleration_due_to_gravity * kilogram / meter**2)
+mmu = mmus = milli_mass_unit = Quantity("milli_mass_unit", mass,
+                                        atomic_mass_unit / 1000)
 quart = quarts = Quantity("quart", length**3, Rational(231, 4) * inch**3)
 
 # Other convenient units and magnitudes
 
-ly = lightyear = lightyears = Quantity("lightyear", length, speed_of_light*julian_year, "ly")
-au = astronomical_unit = astronomical_units = Quantity("astronomical_unit", length, 149597870691*meter, "AU")
+ly = lightyear = lightyears = Quantity("lightyear", length,
+                                       speed_of_light * julian_year, "ly")
+au = astronomical_unit = astronomical_units = Quantity(
+    "astronomical_unit", length, 149597870691 * meter, "AU")
 
 # Planck units:
-planck_mass = Quantity("planck_mass", mass, sqrt(hbar*speed_of_light/G), "m_P")
-planck_time = Quantity("planck_time", time, sqrt(hbar*G/speed_of_light**5), "t_P")
-planck_temperature = Quantity("planck_temperature", temperature, sqrt(hbar*speed_of_light**5/G/boltzmann**2), "T_P")
-planck_length = Quantity("planck_length", length, sqrt(hbar*G/speed_of_light**3), "l_P")
+planck_mass = Quantity("planck_mass", mass,
+                       sqrt(hbar * speed_of_light / G), "m_P")
```

Comment:
It's done by a [YAPF](https://github.com/google/yapf) tool. I can play a bit with weights to prefer one way, but sometimes it is not possible since the expression is too long.